src/mds/Server.cc

```diff
@@ -291,10 +291,10 @@ void Server::dispatch(const cref_t<Message> &m)
   bool sessionclosed_isok = replay_unsafe_with_closed_session;
   // active?
   // handle_peer_request()/handle_client_session() will wait if necessary
+  const auto &req = ref_cast<MClientRequest>(m);
```

Comment:
You can't cast to `MClientRequest` here, the type is not yet known.

Reply:
Okay, will place to its original place then

Reply:
Done